nextflow.config

```diff
@@ -100,6 +100,8 @@ params {
     skip_metabat2                   = false
     skip_maxbin2                    = false
     skip_concoct                    = false
+    bin_domain_classification       = false
+    tiara_min_length                = 3000
```

Comment:
maybe make parameter more explicit: `domain_classification_tiara_min_length` or something, as I guess for some people come into the pipeline might not know exactly what `tiara` is (even if thye are interesting is in domain classification)

Reply:
I disagree because we have also stuff like `freebayes_ploidy` or `pydamage_accuracy` in there that also do not specify what the tool is doing.

Reply:
Fair enough, I guess would require larger overhaul which is out of scope.